cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -82,37 +117,50 @@ public void execute() {
 
             List<SrcFile> sourceFiles = new ArrayList<>();
             List<ModuleMdFile> moduleMdFiles = new ArrayList<>();
+            boolean validInput = false;
+            boolean firstPrompt = true;
             if (interactiveFlag) {
 
-                // Check if Ballerina.toml file needs to be created.
-                out.print("Create Ballerina.toml [yes/y, no/n]: (y) ");
-                String createToml = scanner.nextLine().trim();
-
-                if (createToml.equalsIgnoreCase("yes") || createToml.equalsIgnoreCase("y") || createToml.isEmpty()) {
-                    manifest = new Manifest();
-
-                    String defaultOrg = guessOrgName();
-
-                    String orgName;
-                    do {
-                        out.print("Organization name: (" + defaultOrg + ") ");
-                        orgName = scanner.nextLine().trim();
-                    } while (!validateOrgName(out, orgName));
-                    // Set org-name
-                    manifest.setName(orgName.isEmpty() ? defaultOrg : orgName);
-                    String version;
-                    do {
-                        out.print("Version: (" + DEFAULT_VERSION + ") ");
-                        version = scanner.nextLine().trim();
-                        version = version.isEmpty() ? DEFAULT_VERSION : version;
-                    } while (!validateVersion(out, version));
-
-                    manifest.setVersion(version);
+                if (!manifestExistInProject) {
+                    // Check if Ballerina.toml file needs to be created.
+                    out.print("Create Ballerina.toml [yes/y, no/n]: (y) ");
```

Comment:
it's move everything in this if to separate method, some sections go upto 7 levels of indentation

Reply:
Fixed with a46cc09